app/assets/v2/js/pages/chat.js

```diff
@@ -0,0 +1,12 @@
+
+$(document).ready(function() {
+  $("a").click(function(e){
```

Comment:
this is goiing to block any link on that page, why we just don't comment out the download section?

Reply:
it wont, only ones that have an `#` as their href will have `e.preventDefault()`

Reply:
see the condition on line 5

Reply:
I feel this selector is a bit too global, giving that there are other "#" links

<img width="625" alt="Screen Shot 2020-01-10 at 1 40 42 PM" src="https://user-images.githubusercontent.com/1237259/72173873-ddfea280-33ae-11ea-920b-add24ac09932.png">


There would be more than just the links we want to say coming soon caught under this, selector.

What about doing something like this. 

`$(".my-3 a[href='#']").on('click', (e) => {alert("Coming Soon!"); e.preventDefault(); return;})`

Will only target the anchors inside the my-3 divs section on this page.

Reply:
325e042ff311224688c6ff4adbf86a2e6a72d290